recompress

```diff
@@ -78,7 +78,7 @@ for i in $FILES; do
     COMPRESS="bzip2 -c -"
     NEWFILE="${BASENAME#_service:}.bz2"
   elif [ "$MYCOMPRESSION" == "xz" ]; then
-    COMPRESS="xz -c -"
+    COMPRESS="xz --threads=$(nproc --ignore=1) -c -"
     NEWFILE="${BASENAME#_service:}.xz"
```

Comment:
Setting threads to 0 works (just tested it), and is simpler, it just did not cross my mind at the time. I'll change it.